src/Faker/Provider/Company.php

```diff
@@ -8,6 +8,10 @@ class Company extends \Faker\Provider\Base
         '{{lastName}} {{companySuffix}}',
     );
 
+    protected static $jobTitle = array(
+        'Account Executive', 'Account Manager', 'Accountant', 'Administrative Assistant', 'Assistant Manager', 'Assistant Store Manager', 'Business Analyst', 'Cashier', 'Cook', 'Customer Sales Representative', 'Customer Service', 'Driver', 'Engineer', 'Executive Assistant', 'Financial Analyst', 'Java Developer', 'Maintenance Technician', 'Manager', 'Merchandiser', 'Nurse', 'Occupational Therapist', 'Occupational Therapy Assistant', 'Pharmacist', 'Physcial Therapist', 'Program Manager', 'Project Manager', 'Receptionist', 'Registered Nurse', 'Restaurant Manager', 'Retail Sales Consultant', 'Sales', 'Sales Associate', 'Sales Manager', 'Sales Representative', 'Security Officer', 'Senior Accountant', 'Software Engineer', 'Speech Language Therapist', 'Store Manager', 'Systems Engineer', 'Technician', 'Teller',
```

Comment:
I think this list should be in the en_US locale. For the no-locale formatter, you should rely on the lorem words.

Also, could you please split the line by first letter, to improve code readability?